test/support/CustomMatchers.scala

```diff
@@ -388,7 +393,7 @@ object CustomMatchers {
   }
 
   def haveSelectionForPlaceOfDeposit(expectedSelectOptions: Seq[ExpectedSelectOption]): Matcher[Document] =
-    haveSelectionOptions("placeOfDeposit", "place of deposit", expectedSelectOptions)
+    haveSelectionOptions(FieldNames.placeOfDeposit, "place of deposit", expectedSelectOptions)
```

Comment:
For consistency, do the other fields like "field", "direction" etc need to be replaced similar to above